src/components/LoginForm.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Button, Grid, Link, Stack, Typography, TextField } from '@mui/material';
+const LoginInput = () => {
+  return (
+    <Grid spacing={2} container item direction="column" xs>
+      <Grid item container direction="column" spacing={5} sx={{ textAlign: 'center', alignItems: 'center' }}>
+        <Grid item xs>
+          <Typography variant="h2" fontWeight={'bold'} sx={{ marginLeft: 'auto' }}>
+            Login
+          </Typography>
+        </Grid>
+        <Grid item xs sx={{ width: '70%' }}>
+          <Stack spacing={2}>
+            <TextField type="text" label="Email" />
+            <TextField type="password" label="Password" />
+          </Stack>
+        </Grid>
+        <Grid item xs>
+          <Stack spacing={2}>
+            <Button variant="contained" sx={{ width: '15em' }}>
+              Login
+            </Button>
+            <Link href="#" sx={{ color: 'black', textDecoration: 'underline', fontSize: '1.5em' }}>
+              Forgot Password
+            </Link>
+            <Typography variant="h5" fontWeight="bold">
+              OR
+            </Typography>
+          </Stack>
+        </Grid>
+      </Grid>
+      <Grid item xs>
+        <Stack spacing={2} sx={{ alignItems: 'center' }}>
+          <Button sx={{ bgcolor: '#D9D9D9', color: 'black', '&:hover': { bgcolor: '#D9D9D9' } }}>SIGN IN WITH GOOGLE</Button>
+          <Button sx={{ bgcolor: '#748ADA', color: 'white', '&:hover': { bgcolor: '#748ADA' } }}>SIGN IN WITH DISCORD</Button>
+        </Stack>
```

Comment:
The size of these buttons are too small at 100% size on my end, so the text gets wrapped and it looks very weird. They also don't have the Google and Discord logos, you should add them while you're at it.

![image](https://github.com/LetsMesh/Site/assets/24869207/c2806d6a-7a6c-4d92-839d-a12ac8371a16)

Reply:
@TheRizz mentioned that icons would be added later through an external library and that I could omit them for now.